.github/workflows/ci.yml

```diff
@@ -0,0 +1,107 @@
+name: CI
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - v*
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Commitlint
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: wagoid/commitlint-github-action@v2
+
+  word-blacklist:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Word Blacklist
+        env:
+          BLACKLIST: ${{ secrets.BLACKLIST }}
+        run: ./scripts/ci/word-blacklist.sh
+
+  test:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+
+      - name: Install
+        run: yarn install
+
+      - name: Unit Tests
+        run: yarn test:unit --no-cache --maxWorkers 2 --no-verbose
+
+      - name: Integration Tests
+        run:
+          find packages/demos -iname '*test.ts*' | xargs -n 2 yarn jest --silent
+          --no-cache --no-verbose --no-coverage --maxWorkers 2 --forceExit
+
+  compile:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+
+      - name: Install
+        run: yarn install
+
+      - name: Compile
+        run: yarn compile
+
+      - name: Size Limit
+        run: yarn size-limit
+
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+
+      - name: Install
+        run: yarn install
+
+      - name: Lint
```

Comment:
Is it really faster to do lint/compile/test as individual steps? instead of installing only once and then using npm-run-all -p ?

Reply:
Yes. Remember that the installs are also done in parallel. This also gives us a cleaner output and failure notification compared to `run-p`.

Reply:
Ok